5-WebApp-AuthZ/5-2-Groups/README.md

```diff
@@ -150,34 +235,41 @@ In case, you are authenticating using the [implicit grant flow](https://docs.mic
 
 ### Support in ASP.NET OWIN middleware libraries
```

Comment:
This is ASP.NET Core => not OWIN.
Here this is the OpenIdConnect (Web Apps), or JwtBearer (Web API) middlewares